### ISSUE TITLE
Documentation (or packaging) issue with OkHttp integration

### ISSUE THREAD

**Issue by orrc** [bumptech/glide#532]
According to [the documentation](https://github.com/bumptech/glide/wiki/Integration-Libraries#okhttp), it's possible to use OkHttp by just adding a `compile` line.

However, I found this does not work at all, unless I append `@aar` to the `compile` line, e.g.

    compile 'com.github.bumptech.glide:okhttp-integration:1.3.1@aar'

Without that suffix, the integration classes were included in my APK, but the `<meta-data>` manifest and other resource merging was not done.  Even if I manually added the `<meta-data>` tag, it didn't work.

**Comment by TWiStErRob** [comment]
What's your Grade Android plugin version and what build tools version do you reference inside the `android { }` Gradle extension?

Manually adding it **must** work, otherwise it means that your manifest XML was not compiled properly, try a `clean assemble`.

**Comment by orrc** [comment]
Gradle plugin: 1.2.3
Build tools: 22.0.1

Yeah, I thought it was odd that adding the tag manually didn't work, but I did check the merged manifest and the `<meta-data>` was there, but the OkHttp integration wasn't being called.

I left the `@aar` suffix as it worked, but I can double-check without that suffix and a manual manifest update tomorrow.

**Comment by TWiStErRob** [comment]
Did you have a chance to check it? I played around a little and notice that maybe when you manually added the `<meta-data` tag, you accidentally put it outside the `<application` which for some reason merges, compiles fine, but obviously doesn't work; also IDEA warns about it.

@sjudd As for the `@aar` qualifier, it seems to be required for manifest merging. So @orrc is right.
I tried hard and didn't manage to make it work without it. I think most people (based on what i do) just skim through the wiki and copy-paste *all* code blocks resulting in successful merging because they added the `<meta-data` manually; and I don't remember any other similar report.

I also tried to go back in time by downgrading to Gradle plugin 1.0 and 0.12.2 just to see if `@aar` dependencies were ever implicit and neither of them used OkHttp automagically.
